test/cypress/integration/group-paying.spec.js

```diff
@@ -61,7 +61,7 @@ function openNotificationCard ({
   cy.getByDT('notificationBell').click()
   cy.getByDT('notificationCard').should('be.visible')
 
-  cy.getByDT('notificationCard').within(() => {
+  return cy.getByDT('notificationCard').within(() => {
```

Comment:
Why is this `return` needed?

Reply:
Jut in case the caller wants to be able to continue chaining anything off of this call